packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -53,6 +54,9 @@ let workersConfigsWarningShown = false;
 
 let miniflare: Miniflare | undefined;
 
+/** The inspector port (or false if inspecting is disabled) to use for the vite preview command */
+let previewInspectorPort: number | false | undefined;
```

Comment:
using a variable like this is really ugly.... if anyone has a better suggestion I'm all ears 🥲